### DISCUSSION TITLE
Error in dashboard -site_admin

### DISCUSSION THREAD

**Discussion by goblebun** [LiveHelperChat/livehelperchat#2072] [General]
Hola, estoy actualizando a las 4.34 pero me da este problema 
Gracias de antemano por la ayuda que me podais dar

![image](https://github.com/LiveHelperChat/livehelperchat/assets/888332/95384dd1-4754-4812-9c86-0a3ffe6a40e0)

**Comment by remdex** [comment]
Update properly or hire dev to do it for you... There is docs how to update...
